### PR TITLE
Prepare 0.7rc2

### PR DESCRIPTION
**Pull request by lamblin**

### CODE REVIEW

**Comment by lamblin** [comment]
Fixes #2351

**Comment by lamblin** [comment]
Fixes #2562

File: theano/tensor/basic.py [hunk 1]
```diff
@@ -610,7 +610,7 @@ def get_scalar_constant_value(orig_v, elemwise=True,
                     v = val
                     continue
                 if isinstance(v.owner.op, get_scalar_constant_value_elemwises):
-                    const = [get_scalar_constant_value(i)
+                    const = [numpy.asarray(get_scalar_constant_value(i))
```
Comment (nouiz) [inline theano/tensor/basic.py]: Ok now for this, but it is more tricky from memory.

Comment (lamblin) [inline theano/tensor/basic.py]: I'm removing that since the test case has actually already been fixed.

**Comment by nouiz** [comment]
When travis have at least one section that pass, we can merge.

**Comment by nouiz** [comment]
There is some travis section that passed, so I merge.

**Pull request merged**